staging/src/k8s.io/apiserver/pkg/storage/value/metrics.go

```diff
@@ -41,6 +41,16 @@ var (
 		},
 		[]string{"transformation_type"},
 	)
+	transformerOperationsTotal = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "transformation_operations_total",
+			Help:      "Total number of transformations.",
+		},
+		[]string{"transformation_type", "status"},
+	)
+	// Deprecated, use transformerOperationsTotal instead.
```

Comment:
note the deprecation in the help text so callers are aware?

Reply:
I followed the pattern we used in k8s.io/client-go/util/workqueue/metrics.go, and added deprecated prefix to the variable.
Also added deprecation notice to the help text.

Reply:
Shouldn't the deprecation notice be added to line 59 then?

Reply:
I can go either way.
Tools like Intellij/Goland provide visual indication of the deprecation when the notice is placed at the variable declaration level.

Reply:
Yes, but if it is added to the help it will be output in '/metrics' endpoint which is kinda too nice, imo.